google/ggrc-core
# GGRC-5286 Requests are canceled after Workflow creation via "create" modal

# Issue description

Requests are canceled after Workflow creation via "create" modal.

# Steps to test the changes

Steps to reproduce:
1. Open LHN -> Workflow.
2. Click "create" link.
3. Open Dev Tools -> Network tab -> Switch "Preserve log" to true.
4. Click "Save & Close" button placed on filled Workflow modal.

Actual result: There are canceled requests.
Expected result: There are no canceled requests.

On local machine:
![image](https://user-images.githubusercontent.com/13063442/41396553-0234db98-6fba-11e8-98bd-9e4e24eaa767.png)

On ggrc-test:
![image](https://user-images.githubusercontent.com/13063442/41396588-1fcde334-6fba-11e8-919e-4ec248beaabf.png)

# Solution description

Canceled requests are redundant in case, when some workflow is created via LHN. They are necessary only, when we create/clone Task Group within Workflow's info page. Thus, these requests were moved to places, where they are necessary.

Must be checked the following points:
1. Create TG from WF page (Add & Save + Save & Add another);
2. Clone TG from WF page via 3 dots menu within any TG info panel.
3. Approve any object;
4. Create WF via LHN;
5. Clone WF ( WF's info page -> 3 dots menu -> clone workflow).
# Sanity checklist




# PR Review checklist
